File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/J. Primes from 1 to n/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/J
#include <iostream>
#include <cmath>
using namespace std;
 
int main()
{
    unsigned int num;
    bool isPrime = true;
    cin >> num;
    for (int n = 2; n <= num; n++)
    {
        for (int i = 2; i <= sqrt(n); i++)
        {
            if (n % i == 0)
            {
                isPrime = false;
                break;
            }
            isPrime = true;
        }
        if (isPrime)
        {
            cout << n << ' ';
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/J. Primes from 1 to n/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/J

using System;
using static System.Math;

namespace Codeforces.Assuit.Contest1
{
    class ProblemJ
    {
        static void Main(string[] args)
        {
            uint num = uint.Parse(Console.ReadLine());
            bool isPrime = true;

            for (int n = 2; n <= num; n++)
            {
                for (int i = 2; i <= Sqrt(n); i++)
                {
                    if (n % i == 0)
                    {
                        isPrime = false;
                        break;
                    }
                    isPrime = true;
                }
                if (isPrime)
                {
                    Console.Write("{0} ", arg0:n);
                }
            }
        }
    }
}
